assets/js/items.js

```diff
@@ -112,35 +116,27 @@ class Item extends BaseItem {
             --><div class="counter-button">+</div>
           </span>
         </span>
-      </div>
-    `).translate();
-    this.$menuButton[0].rdoItem = this;
+    `;
+    Language.translateDom(menuBtn);
+    this.menuButton =menuBtn;
+    this.menuButton.rdoItem = this;
     this.amount = this.amount; // trigger counter update
-    this.$menuButton
-      .appendTo(this.collection.$submenu)
+    this.collection.submenu.appendChild(this.menuButton);
     Loader.mapModelLoaded.then(() => {
-      SettingProxy.addListener(InventorySettings, 'isEnabled', () =>
-        this.$menuButton
-          .find('.counter')
-          .toggle(InventorySettings.isEnabled)
-          .end()
-      )();
+      SettingProxy.addListener(InventorySettings, 'isEnabled', () => {
+        this.menuButton.querySelector('.counter').style.display = InventorySettings.isEnabled ? '' : 'none';
+      })();
     });
   }
   get amount() {
     return +localStorage.getItem(this._amountKey);
   }
   set amount(value) {
-    if (value < 0) value = 0;
-    if (value) {
-      localStorage.setItem(this._amountKey, value);
-    } else {
-      localStorage.removeItem(this._amountKey);
-    }
-    this.$menuButton.add(this.$weeklyMenuButton)
-      .find('.counter-number')
-      .text(value)
-      .toggleClass('text-danger', value >= InventorySettings.stackSize);
+    value = value < 0 ? 0 : value;
+    value ? localStorage.setItem(this._amountKey, value) : localStorage.removeItem(this._amountKey);
+    [this.menuButton.querySelector('.counter-number'), this.weeklyMenuButton && this.weeklyMenuButton.querySelector('.counter-number')].forEach(btn => {
+      btn && (btn.textContent = value, btn.classList.toggle('text-danger', value >= InventorySettings.stackSize));
```

Comment:
This would be much more readable.
Also this comma syntax is sensitive to wrapping brackets. I saw it in a few spots. Please use an if on the beginning, and then each statement separated by a new line. Correct like that all that comma separated functions/statements.

```javascript
if (!btn) return;
btn.textContent = value;
btn.classList.toggle('text-danger', value >= InventorySettings.stackSize);
```